modules/openapi-generator/src/main/resources/swift5/Extensions.mustache

```diff
@@ -206,3 +206,13 @@ extension Set: RequestDecodable where Element: Content {
 extension Set: Content where Element: Content { }
 
 extension AnyCodable: Content {}{{/useVapor}}
+
+extension JSONEncodable where Self: Encodable {
```

Comment:
@0x0c could you please move this change to line 63, so that it's not generated when using vapor, so that CI passes?

Reply:
Done with https://github.com/OpenAPITools/openapi-generator/pull/11202/commits/ebd7d017bbb6c7ccf986914b403b5ec06a32bab4